csharp/generate_protos.sh

```diff
@@ -26,7 +26,12 @@ if [ -z "$PROTOC" ]; then
 fi
 
 # Descriptor proto
-#TODO(jtattermusch): generate descriptor.proto
+# Temporary manual fixup...
```

Comment:
I'd say putting a comment that describes exacly what this temporary fixup is doing would be useful (basically just pasting the text from PR description here).